src/components/events/EventDialog.jsx

```diff
@@ -46,18 +46,25 @@ class EventDialog extends React.Component {
         onClick={this.handleClickOpen}
         onClose={this.handleClose}>
         <DialogHeader collectionName="events" docId={eventId} title={`${dialogTitle} Event`} />
-        <EventForm {...this.props} onModalClose={this.handleClose} />
+        <EventForm
+          currentValues={currentValues}
+          eventId={eventId}
+          formType={formType}
+          onModalClose={this.handleClose} />
       </DFDialog>
     );
   }
 }
 
 EventDialog.propTypes = {
-  currentValues: PropTypes.shape().isRequired,
+  currentValues: PropTypes.shape(),
```

Comment:
If we're doing this .shape() function, this may be a point where we can start bringing in stronger assurances on our PropTypes. 

Ref: https://reactjs.org/docs/typechecking-with-proptypes.html, take a look at `optionalObjectWithShape `. For these shapes, should we add in the actual structure of currentValues if we have them? Seems like we already know what we're passing down.